src/classes/UTIL_MasterSchedulableHelper.cls

```diff
@@ -124,7 +124,11 @@ public without sharing class UTIL_MasterSchedulableHelper {
     public static void setScheduledJobs() {
 
         Map<String, CronTrigger> scheduledJobs = new Map<String, CronTrigger>();
-        
+
+        // always delete and reschedule all jobs in this particular scenario
+        Error_Settings__c orgErrorSettings = UTIL_CustomSettingsFacade.getOrgErrorSettings();
+        Boolean rescheduleAllJobs = (UTIL_Namespace.isSandbox() && orgErrorSettings.Sandbox_Jobs_Rescheduled__c == false);
```

Comment:
no I just mean to grab the rest of the conditions from your if statement so you can just state if (rescheduleAllJobs) vs if (rescheduleAllJobs || this || that, so put this or that logic in your variable.